test/lib/pgBackRestTest/Common/ContainerTest.pm

```diff
@@ -488,10 +495,16 @@ sub containerBuild
                 if (vmPgRepo($strVm))
                 {
                     $strScript .=
-                        "    echo \"deb http://apt" . ($strVm eq VM_U20 ? '-archive' : '') . ".postgresql.org/pub/repos/apt/ \$(lsb_release -s -c)-pgdg main" .
-                            "\" >> /etc/apt/sources.list.d/pgdg.list && \\\n" .
-                        "    wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - && \\\n" .
-                        "    apt-get update && \\\n";
+                      #  "    apt-get install -y --no-install-recommends postgresql-common && \\\n" .
+                      #  "    /usr/share/postgresql-common/pgdg/apt.postgresql.org.sh -y" .
+                         "    cat /etc/os-release && \\\n" .
+                         "    apt install curl ca-certificates && \\\n" .
+                         "    install -d /usr/share/postgresql-common/pgdg && \\\n" .
+                         "    curl -o /usr/share/postgresql-common/pgdg/apt.postgresql.org.asc --fail https://www.postgresql.org/media/keys/ACCC4CF8.asc && \\\n" .
+                         "    echo \"deb https://apt-archive.postgresql.org/pub/repos/apt/ jammy-pgdg-archive main \" > /etc/apt/sources.list.d/pgdg.list && \\\n" .
```

Comment:
You miss the key: `deb [signed-by=/usr/share/postgresql-common/pgdg/apt.postgresql.org.asc] https://apt-archive.postgresql.org/pub/repos/apt/ jammy-pgdg-archive main`